packages/ui/Input/Select.tsx

```diff
@@ -46,12 +49,12 @@ function useSelectContext<T>() {
 
 // SelectRoot 컴포넌트: Select 컴포넌트에게 context를 제공
 function SelectRoot<T extends string | number | boolean>(props: SelectProps<T>) {
-  const { children, onChange, defaultValue, type, visibleOptions = 5, className } = props;
+  const { children, onChange, defaultValue, type, visibleOptions = 5, className, multiple } = props;
 
   const buttonRef = useRef<HTMLButtonElement>(null);
   const optionsRef = useRef<HTMLUListElement>(null);
 
-  const [selected, setSelected] = useState<Option<T> | null>(defaultValue ?? null);
+  const [selected, setSelected] = useState<Option<T> | Option<T>[] | null>(defaultValue ?? null);
```

Comment:
`multiple`을 도입한 시점에서 Option과 Option[]을 구분하지 않고, 그냥 어레이로 타입을 설정한 후에, multiple이 아니라면 [0]으로 접근하면 되지 않을까요 ?

컴포넌트 내 함수들에서 해당 타입에 대한 분기처리 때문에 불필요한 강제 캐스팅이 많아지는 것 같아요. 그냥 배열로 설정한다면 많이 해소될 것 같아요.

다만 `onChange`와 같은 기존에 존재하던 인터페이스에서는 value를 `T[]`로 좁히게 되면 영향 범위가 있을 수도 있으니, 유지해야할 것 같아요.

```suggestion
  const [selected, setSelected] = useState<Option<T>[] | null>(defaultValue ? [defaultValue] : null);
```

Reply:
감사합니다. 타입 단언이 필요한 경우도 해소할 수 있겠네요! 
추가 사항( label 세팅 하지 않으면 첫번째로 선택한 값이 label 로 설정됨 -> 스토리북에도 작성 ) 과 함께 수정해서 푸시 했습니다!